### PR TITLE
Master updates

### PR DESCRIPTION
**Pull request by rashidfawad**
I have updated Symfony from 2.3 to 2.8 along with some other dependencies. I have also removed some deprecations. The November Camp page route did not have any Controller specified.  Perhaps we can fix it after you review the changes. 

Cheers

### CODE REVIEW

**Review by Nyholm — CHANGES_REQUESTED** [review]
Awesome. Thank you. Ive reviewed this PR and tested it. I like it. I only had a few comments. 

A question, Why did you change all the file permissions?

**Comment by rashidfawad** [comment]
Than you for review. I have added Twig/Extensions, replaced tilde with caret in composer. I could use FrameworkBundle TemplateAction for rendering the november-camp page but then I did not have the template so I skipped it. As each page is built using small text portions.
Regarding the file permissions usually when I need to change multiple files, I usually do "sudo chmod -R 777 abc" to make it editable :).

**Comment by Nyholm** [comment]
Thank you. 
If you revert the file permissions, I'll be happy to merge. We do not need the world to be able to edit the source.

**Comment by rashidfawad** [comment]
Hi, I can't change the permissions to 644 because when I change it I can no longer use git to commit the changes. I also could not use symfony.se the first time I cloned it from Github. I had to change the permission then I could access the site on localhost. 

If you can change the permissions, kindly do it and then merge it. Thank you.

File: composer.json [hunk 1]
```diff
@@ -4,49 +4,58 @@
...
-        "knplabs/knp-markdown-bundle": "1.2.*",
-        "dms/meetup-api-bundle": "~1.0",
-        "happyr/slugify-bundle": "1.0.*",
-        "debril/rss-atom-bundle": "~1.3",
-        "happyr/excerpt-bundle": "0.1.*"
+        "php": ">=5.3.9",
+        "symfony/symfony": "2.8.*",
+        "doctrine/orm": "^2.4.8",
+        "doctrine/doctrine-bundle": "^1.4",
+        "twig/twig": "^1.0||^2.0",
```
Comment (Nyholm) [inline composer.json]: Why do we want to support both?
Also, it fails under twig 2.0

File: composer.json [hunk 2]
```diff
@@ -4,49 +4,58 @@
...
+        "symfony/assetic-bundle": "^2.3",
+        "symfony/swiftmailer-bundle": "^2.3,>=2.3.10",
+        "symfony/monolog-bundle": "^3.0.2",
+        "sensio/distribution-bundle": "^4.0",
+        "sensio/framework-extra-bundle": "^3.0.2",
+        "incenteev/composer-parameter-handler": "^2.0",
+        "knplabs/knp-markdown-bundle": "^1.5.1",
+        "dms/meetup-api-bundle": "^1.0.0",
+        "happyr/slugify-bundle": "^1.0.1",
+        "debril/rss-atom-bundle": "^3.0.5",
```
Comment (Nyholm) [inline composer.json]: What were the BC breaks between 1.3.2 and ^3.0.5? I can see some that has not been taken care of.

**Review by Nyholm — COMMENTED** [review]
There were some issues and fixed that has not been completed here. 

Ie, the test code did not run. I have fixed those and created a new PR.

**Comment by Nyholm** [comment]
Ive merged the changes. Thank you for this PR.

**Comment by rashidfawad** [comment]
I can't access the site. Can you check the logs on the server where the problem is? It's giving me 500 error.

File: composer.json (hunk 1 above)
Comment (rashidfawad) [inline composer.json]: No, we don't have to support both. We should update it to the latest if we want to switch to Symfony3.  If there is some deprecation warning or BC break by using version 1.0. I can remove it. It was working for me on localhost.

**Comment by Nyholm** [comment]
FIxed now. I added PHP7 and HTTPS

**Comment by rashidfawad** [comment]
Yuup, saw it in the commit. The error may have occurred because we were referencing Http links from Https.

**Pull request merged**